gradle/baseline.gradle

```diff
@@ -1,42 +0,0 @@
-apply plugin: 'com.palantir.baseline-checkstyle'
-
-// ./gradlew compileJava compileTestJava -PenableErrorProne=true
-if (enableErrorProne.toBoolean()) {
```

Comment:
moved all of this to `build.gradle` and this is no longer opt-in, its default on everywhere

Reply:
So reading through it looks like this was doing some per main/test package differences, but I think it's fine. We should quickly drive all the turned off warnings.